pkg/syncer/resource_syncer.go

```diff
@@ -40,11 +40,15 @@ import (
 	"k8s.io/klog"
 )
 
+const OrigNamespaceLabelKey = "submariner-io/originatingNamespace"
```

Comment:
We have something similar that we use in lighthouse for namespace as well as cluster. In a separate Pr we should have admiral tag the sourceCluster too.

Reply:
yes we can.